agent/ioproxy_handler.go

```diff
@@ -16,11 +16,11 @@ package main
 import (
 	"context"
 
+	task "github.com/containerd/containerd/api/runtime/task/v2"
 	"github.com/containerd/containerd/log"
-	"github.com/containerd/containerd/runtime/v2/task"
 	"github.com/firecracker-microvm/firecracker-containerd/internal/vm"
 	ioproxy "github.com/firecracker-microvm/firecracker-containerd/proto/service/ioproxy/ttrpc"
-	"github.com/gogo/protobuf/types"
+	"google.golang.org/protobuf/types/known/emptypb"
```

Comment:
containerd has defined a [types](https://github.com/containerd/containerd/blob/main/protobuf/types/types.go) package to make this migration easier. This should also help with any future migration.

Reply:
Updated. @swagatbora90 PTAL